lib/rock/webapp/tasks/api.rb

```diff
@@ -8,6 +9,12 @@ class API < Grape::API
                 version 'v1', using: :header, vendor: :rock
                 format :json
 
+                if ENV["REST_API_LOG_CONSOLE"]
+                    puts "logging"
```

Comment:
Sorry I missed the updates.

Why not use the logger to display "logging". Moreover, you should really make the message less cryptic, remember that this can be loaded by other applications. E.g. "enabled rest_api logging to console, set REST_API_LOG_CONSOLE to anything but '1' to disable"